force-app/test/ADDR_Addresses_TEST.cls

```diff
@@ -2073,8 +2073,7 @@ public with sharing class ADDR_Addresses_TEST {
     **********************************************************************************************************/
     @IsTest
     private static void addressShouldBePropagatedToAccountsAndContactsOnMultipleTriggerCalls() {
-        // Arrange
-        Integer totalHH = 210;
```

Comment:
Are we losing a test scenario here (multiple trigger calls), or is this no longer needed now that we removed the TDTM toggles?

Reply:
I actually think @lparrott has a good point looking back at this. The intent was to max the default collection of trigger context records before a second trigger call is required (200). I'll revert this.